pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -188,13 +189,14 @@ type osqueryOptions struct {
 	stdout              io.Writer
 }
 
-func newInstance(knapsack types.Knapsack, serviceClient service.KolideService, opts ...OsqueryInstanceOption) *OsqueryInstance {
+func newInstance(registrationId string, knapsack types.Knapsack, serviceClient service.KolideService, opts ...OsqueryInstanceOption) *OsqueryInstance {
 	runId := ulid.New()
 	i := &OsqueryInstance{
-		knapsack:      knapsack,
-		slogger:       knapsack.Slogger().With("component", "osquery_instance", "instance_run_id", runId),
-		serviceClient: serviceClient,
-		runId:         runId,
+		registrationId: registrationId,
+		knapsack:       knapsack,
+		slogger:        knapsack.Slogger().With("component", "osquery_instance", "instance_run_id", runId),
```

Comment:
do you think it's worth adding the `registrationId` to slogger here as well?

Reply:
Sure, let me do that in the next PR (https://github.com/kolide/launcher/pull/1941)

Reply:
Done: https://github.com/kolide/launcher/pull/1941/commits/b3e2ee91c443c3c5623418b50df615b6e5a21f67